syspurpose/README.md

```diff
@@ -10,9 +10,5 @@ The recommended workflow is as follows:
 1) Run the following to install project deps in the virtualenv: `pipenv install`
```

Comment:
In case you use your system for development of subman using Python 2.7, then it is recommended to `unset PYTHONPATH`. Otherwise you can get following error:

```
(syspurpose-HshUz_dS) [jhnidek@fedora syspurpose]$ syspurpose --help
Traceback (most recent call last):
  File "/home/jhnidek/.local/share/virtualenvs/syspurpose-HshUz_dS/bin/syspurpose", line 11, in <module>
    load_entry_point('syspurpose==1.22.1', 'console_scripts', 'syspurpose')()
  File "/home/jhnidek/.local/share/virtualenvs/syspurpose-HshUz_dS/lib/python3.6/site-packages/pkg_resources/__init__.py", line 479, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/jhnidek/.local/share/virtualenvs/syspurpose-HshUz_dS/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2703, in load_entry_point
    return ep.load()
  File "/home/jhnidek/.local/share/virtualenvs/syspurpose-HshUz_dS/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2321, in load
    return self.resolve()
  File "/home/jhnidek/.local/share/virtualenvs/syspurpose-HshUz_dS/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2327, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
ImportError: bad magic number in 'syspurpose': b'\x03\xf3\r\n'
```

There can be some Python 2.7 `*.pyc` files causing this error.

Reply:
I didn't write the README.  The only reason it's in this commit is because I moved it.  Can you just submit a PR after this is merged to address this particular issue?

Reply:
Sounds good.